### PR TITLE
week-6 assignments added

### PR DESCRIPTION
**Pull request by Bharat2044**
1. Auth App
2. Finance Tracker App
3. Todo App

### CODE REVIEW

**Comment by Bharat2044** [comment]
@hkirat sir please review this assignments problems

File: week-6/Auth-App/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# **Authentication App with Signup Signin and Logout Features**
+
+Create a complete authentication application using Express for the backend and JavaScript for the frontend. The application will support user signup, signin, and logout features.
```
Comment (VineeTagarwaL-code) [inline week-6/Auth-App/README.md:3]: feels like grammar can be improved here

File: week-6/Finance-Tracker-App/backend/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    
+});
+
+// Add transaction route
+app.post('/transactions', authenticate, (req, res) => {
+    
+});
+
+// Update transaction route
+app.put('/transactions/:id', authenticate, (req, res) => {
```
Comment (VineeTagarwaL-code) [inline week-6/Finance-Tracker-App/backend/index.js:47]: use PATCH method, it makes more sense cause we are updating it partially

File: week-6/Todo-App/backend/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    
+});
+
+// Route to delete a To-Do
+app.delete("/todos/:id", auth, function(req, res) {
+    
+});
+
+// Route to mark a To-Do as done/undone using PUT
+app.put("/todos/:id/done", auth, function(req, res) {
```
Comment (VineeTagarwaL-code) [inline week-6/Todo-App/backend/index.js:63]: patch method please

**Review by VineeTagarwaL-code — CHANGES_REQUESTED** [review]
i have requested the needed changes, thank you for your contribution !

File: week-6/Finance-Tracker-App/README.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ Create a Personal Finance Tracker application using Express for the backend and
     - **POST `/transactions`**:
         - Create new transactions with details such as amount, type, category, and date.
 
-    - **PUT `/transactions/:id`**:
+    - **PATCH/PUT `/transactions/:id`**:
```
Comment (VineeTagarwaL-code) [inline week-6/Finance-Tracker-App/README.md:60]: just patch

File: week-6/Todo-App/README.md [hunk 5]
```diff
@@ -49,8 +49,8 @@ Create a complete Todo application using Express for the backend and JavaScript
 4. Implement middleware to validate JWT tokens for protected routes.
 5. Implement the `/todos` GET route to return all todos data.
 6. Implement the `/todos` POST route to create new todos data with id, username, title and done.
-7. Implement the `/todos/:id` PUT route to update todos title for a specific id.
-8. Implement the `/todos/:id/done` PUT route update todos done field true/false based on incoming request.
+7. Implement the `/todos/:id` PATCH/PUT route to update todos title for a specific id.
```
Comment (VineeTagarwaL-code) [inline week-6/Todo-App/README.md:52]: just patch

File: week-6/Todo-App/backend/index.js (hunk 3 above)
Comment (Bharat2044) [reply week-6/Todo-App/backend/index.js]: @VineeTagarwaL-code Added PATCH method

**Review by VineeTagarwaL-code — CHANGES_REQUESTED** [review]

File: week-6/Todo-App/README.md (hunk 5 above)
Comment (Bharat2044) [reply week-6/Todo-App/README.md]: updated

**Pull request closed without merging**